### PR TITLE
Do not unsubscribe from the bus when there is no UI

### PR DESCRIPTION
**Pull request by wagoodman**
This PR changes the behavior of the event loop to not unsubscribe from the bus when there is no UI.

Additionally:
- tweaks the output of the --help text
- changes the profile enum constants
- adds field descriptors to configs

### CODE REVIEW

File: logging.go [hunk 1]
```diff
@@ -54,19 +54,22 @@ var _ LoggerConstructor = DefaultLogger
...
 	// Structured   bool         `yaml:"structured" json:"structured" mapstructure:"structured"`                        // show all log entries as JSON formatted strings
 }
 
-var _ fangs.FlagAdder = (*LoggingConfig)(nil)
-var _ fangs.PostLoad = (*LoggingConfig)(nil)
+var _ interface {
+	fangs.PostLoader
+	fangs.FlagAdder
+	fangs.FieldDescriber
+} = (*LoggingConfig)(nil)
```
Comment (kzantow) [inline logging.go:72]: 😎

File: application.go [hunk 2]
```diff
@@ -214,7 +215,7 @@ func (a *application) SetupCommand(cmd *cobra.Command, cfgs ...any) *cobra.Comma
 }
 
 func (a *application) SetupPersistentCommand(cmd *cobra.Command, cfgs ...any) *cobra.Command {
-	return a.setupCommand(cmd, cmd.PersistentFlags(), &cmd.PersistentPreRunE, cfgs...)
+	return a.setupCommand(cmd, cmd.PersistentFlags(), &cmd.PreRunE, cfgs...)
```
Comment (kzantow) [inline application.go:218]: I don't quite understand this change -- was it causing setup to be run multiple times for subcommands?

**Review by kzantow — APPROVED** [review]
LGTM

File: application.go (hunk 2 above)
Comment (wagoodman) [reply application.go:218]: yup yup 👍

**Pull request merged**